internal/chain/chain.go

```diff
@@ -20,28 +19,13 @@ import (
 )
 
 type ChainOpts struct {
-	ChainUrl       string
-	ChainAuthToken string
-	ChainPk        string
-	NaAddress      common.Address
-	VpaAddress     common.Address
-	CaAddress      common.Address
-}
-
-func InitializeEthChainService(chainOpts ChainOpts) (*chainservice.EthChainService, error) {
```

Comment:
This function is no longer needed since the `chainservice.NewEthChainService` now accepts `chainOpts` as input.